redaxo/src/addons/cronjob/lib/command/run.php

```diff
@@ -35,4 +47,48 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io->success('Cronjobs skipped.');
         return 0;
     }
+
+    private function executeSingleJob(SymfonyStyle $io, $id)
+    {
+        $manager = rex_cronjob_manager_sql::factory();
+
+        if (null === $id) {
+            $jobs = rex_sql::factory()->getArray('
+                SELECT id, name
+                FROM ' . REX_CRONJOB_TABLE . '
+                WHERE environment LIKE "%|script|%"
+                ORDER BY id
+            ');
+            $jobs = array_column($jobs, 'name', 'id');
+
+            $question = new ChoiceQuestion('Which cronjob should be executed?', $jobs);
+            $question->setValidator(function ($selected) use ($jobs) {
+                $selected = trim($selected);
+
+                if (!isset($jobs[$selected])) {
+                    throw new SymfonyInvalidArgumentException(sprintf('Value "%s" is invalid.', $selected));
+                }
+
+                return $selected;
+            });
+
+            $id = $io->askQuestion($question);
+            $name = $jobs[$id];
+        } else {
+            $name = $manager->getName($id);
+        }
+
+        $success = $manager->tryExecute($id);
+
+        $msg = '';
+        if ($manager->hasMessage()) {
+            $msg = ': '.$manager->getMessage();
```

Comment:
DecodeMessage() ?

Reply:
Bin mir unsicher, ob die Message überhaupt in HTML ist bzw. sein sollte.
Die Message landet ja normalerweise in der Log-Datei.
Die Log-Ausgabe sieht allerdings so aus: https://github.com/redaxo/redaxo/blob/master/redaxo/src/addons/cronjob/pages/log.php#L42
- einerseits ersetzen wir dort `\n` durch `<br>`
- andererseits nutzen wir aber kein htmlspecialchars

Weiß nicht, ob das da ein Bug ist, oder ob wir bewusst HTML in den Messages zulassen.